docs/forwarding-events-to-destinations/forwarding-events/google-tag-manager-server-side/snowplow-tag-for-gtm-ss/snowplow-tag-configuration/index.md

```diff
@@ -62,28 +95,220 @@ A table of definitions for self-describing data properties. Each row maps a sing
 
 For each Event, you can also read properties off the client event object and add them as properties to the Self Describing Event.
 
-![](images/sdjsetup.png)
+### Context entities
```

Comment:
This context entities section is _massive_. It dwarfs the page in many ways for an "advanced" feature. I wonder if we should move it to a subpage - we could still link to it half way down this page here but also it'll be in the nav then.

Maybe move all the Advanced Event Settings into a sub page?

Reply:
Done! It looks nicer now and also more headers are visible in the table of contents, since they are a level up in the new sub page!
I am wondering whether to keep the subheaders of the Advanced Event Settings section in the main configuration page as well (for any links to headers). Would that be better?